error-prone-guidelines/pom.xml

```
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>tech.picnic.error-prone-support</groupId>
+        <artifactId>error-prone-support</artifactId>
+        <version>0.14.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>error-prone-guidelines</artifactId>
+
+    <name>Picnic :: Error Prone Support :: Error Prone Guidelines</name>
+    <description>Error Prone checks defining coding guidelines for the Error Prone project.</description>
```

```suggestion
    <description>Coding guidelines for the Error Prone Support project.</description>
```
(This to (a) prepare for moving Refaster rules here and (b) because effectively these rules only apply to this project.)